source/isaaclab_mimic/isaaclab_mimic/envs/__init__.py

```diff
@@ -16,13 +16,6 @@
 from .franka_stack_ik_rel_skillgen_env_cfg import FrankaCubeStackIKRelSkillgenEnvCfg
 from .franka_stack_ik_rel_visuomotor_cosmos_mimic_env_cfg import FrankaCubeStackIKRelVisuomotorCosmosMimicEnvCfg
 from .franka_stack_ik_rel_visuomotor_mimic_env_cfg import FrankaCubeStackIKRelVisuomotorMimicEnvCfg
```

Comment:
Thank you for the fix! Can we also propagate the changes to these imports here? We shouldn't be importing the env cfgs since they increase the module load time.